### ISSUE TITLE
Proposal to create timers using the current branch name.

### ISSUE THREAD

**Issue by raphaelkieling** [remotemobprogramming/mob#343]
Hello folks.

Something that I'm missing is an easy way to start a timer link without the need to explicitly change the `.mob` file with the `MOB_TIMER_ROOM` flag. I tried:
- `export MOB_TIMER_ROOM="my-test" mob start`
- `MOB_TIMER_ROOM="my-test" mob start`
- `export MOB_TIMER_ROOM="my-test" && mob start`
- `MOB_TIMER_ROOM="my-test-2" mob start --create x`
   - For some reason, it works haha, but it isn't good to remember and share with the team. Another thing is that is totally necessary to send a name to the flag, in this case, `x`.
   
It means that the flag is being ignored in my vision. Is it right?

### Proposal

Maybe something like `mob start --create-timer-current-branch`(`mob start -ct` is just an example) would be perfect. It will avoid the necessity to put the name of the room on each dev `.mob` file. 

- option 1:
```
Dev A
- git checkout -b my-feature
- mob start 10 -ct
   - `mob` will automatically create a timer `https://timer.mob.sh/my-feature`
- mob next -ct

Dev B:
- mob start 10 -ct
   - will send the new person to `https://timer.mob.sh/my-feature`
- mob break 5 -ct
  - will the break time to`https://timer.mob.sh/my-feature`
```

- option 2
```
export CREATE_TIMER_USING_CURRENT_BRANCH=true
The same as OPTION 1 but avoiding sending the ct flag.
```

In summary, it will use the current branch name for everything related to the timer. Is it make sense to you?

**Comment by simonharrer** [comment]
Have you tried out the parameter MOB_TIMER_ROOM_USE_WIP_BRANCH_QUALIFIER set to true in your config? It should do what you want. Simply set it in your config .mob file in your home or project folder.

**Comment by raphaelkieling** [comment]
Thanks for the quickly response, yes i did:

1 - Change `MOB_TIMER_ROOM_USE_WIP_BRANCH_QUALIFIER` to `true` inside the `.mob`
2 - Run `mob start`
3 - I can't see any timer link.

**Comment by hollesse** [comment]
Hey @raphaelkieling,
to use the branch qualifier in mob you need to set it. Therefore use `mob start -b yourBranchQualifier` instead of just `mob start` then it should work as you expected it ;)
